src/Evolution/Systems/GeneralizedHarmonic/DampedHarmonicGaugeQuantities.cpp

```diff
@@ -0,0 +1,928 @@
+  spacetime_deriv_of_log_factor(                                               \
+    const Scalar<DTYPE(data)>& lapse,                                          \
+    const tnsr::I<DTYPE(data), DIM(data), FRAME(data)>& shift,                 \
+    const tnsr::A<DTYPE(data), DIM(data), FRAME(data)>& spacetime_unit_normal, \
+    const tnsr::II<DTYPE(data),DIM(data), FRAME(data)>& inverse_spatial_metric,\
+    const Scalar<DTYPE(data)>& sqrt_det_spatial_metric,                        \
+    const tnsr::ii<DTYPE(data), DIM(data), FRAME(data)>& dt_spatial_metric,    \
+    const tnsr::aa<DTYPE(data), DIM(data), FRAME(data)>& pi,                   \
+    const tnsr::iaa<DTYPE(data), DIM(data), FRAME(data)>& phi,                 \
+    const double& exponent) noexcept;                                          \
+  template tnsr::a<DTYPE(data), DIM(data), FRAME(data)>                        \
+  GeneralizedHarmonic::DampedHarmonicGaugeHelpers::                            \
+  spacetime_deriv_of_power_log_factor(                                         \
+    const Scalar<DTYPE(data)>& lapse,                                          \
+    const tnsr::I<DTYPE(data), DIM(data), FRAME(data)>& shift,                 \
+    const tnsr::A<DTYPE(data), DIM(data), FRAME(data)>& spacetime_unit_normal, \
+    const tnsr::II<DTYPE(data),DIM(data), FRAME(data)>& inverse_spatial_metric,\
+    const Scalar<DTYPE(data)>& sqrt_det_spatial_metric,                        \
+    const tnsr::ii<DTYPE(data), DIM(data), FRAME(data)>& dt_spatial_metric,    \
+    const tnsr::aa<DTYPE(data), DIM(data), FRAME(data)>& pi,                   \
+    const tnsr::iaa<DTYPE(data), DIM(data), FRAME(data)>& phi,                 \
+    const double& g_exponent, const double& exponent) noexcept;
+
+#define INSTANTIATE_SCALAR_FUNC(_, data)                                       \
+  template Scalar<DTYPE(data)>                                                 \
+  GeneralizedHarmonic::DampedHarmonicGaugeHelpers::log_factor(                 \
+    const Scalar<DTYPE(data)>& lapse,                                          \
+    const Scalar<DTYPE(data)>& sqrt_det_spatial_metric,                        \
+    const double& exponent) noexcept;
+
+
+#define INSTANTIATE_CI(_, data)                                                \
+  template struct GeneralizedHarmonic::DampedHarmonicHCompute<DIM(data),       \
+      FRAME(data)>;                                                            \
+  template struct                                                              \
+    GeneralizedHarmonic::SpacetimeDerivDampedHarmonicHCompute<DIM(data),       \
+      FRAME(data)>;
+
+GENERATE_INSTANTIATIONS(INSTANTIATE, (1, 2, 3), (double, DataVector),
+                        (Frame::Grid, Frame::Inertial))
+
+GENERATE_INSTANTIATIONS(INSTANTIATE_SCALAR_FUNC, (3), (double, DataVector),
+                        (Frame::Inertial))
```

Comment:
Because those `SCALAR_FUNC`s do not depend on either Frame or Dimension. Both `Frame::Inertial` and `Frame::Grid` instantiations would be identical to each other, which wont compile. This is also why they're only instantiated for `DIM=3` and not for `DIM=(1,2,3)`.